telcell/models/simplemodel.py

```diff
@@ -0,0 +1,146 @@
+import datetime
+from typing import List
+from collections import defaultdict
+
+from telcell.data.models import Measurement, Track, MeasurementPair
+
+
+def get_measurement_with_minimum_time_difference(track: Track,
```

Comment:
Oude comment van Jeroen: Een logischer argument lijkt me measurements: Iterable[Measurement], dan mag het namelijk nog steeds een Track zijn (dat is namelijk een Iterable[Measurement]), maar het zou bijv. ook een List[Measurement] oid mogen zijn. Je doet hier immers niets met het feit dat het een Track is (zoals de owner of de name), je gebruikt alleen de measurements.